### PR TITLE
Allow for adding prefix in manchester notation.

### PR DESCRIPTION
**Pull request by francescalb**
# Description:

## Type of change:

- [ ] Bug fix.
- [x] New feature.
- [ ] Documentation update.

## Checklist:

This checklist can be used as a help for the reviewer.

- [ ] Is the code easy to read and understand?
- [ ] Are comments for humans to read, not computers to disregard?
- [x] Does a new feature has an accompanying new test (in the CI or unit testing schemes)?
- [ ] Has the documentation been updated as necessary?
- [ ] Does this close the issue?
- [ ] Is the change limited to the issue?
- [ ] Are errors handled for all outcomes?
- [ ] Does the new feature provide new restrictions on dependencies, and if so is this documented?

### CODE REVIEW

File: tests/testonto/excelparser/fromexcelonto.ttl [hunk 1]
```diff
@@ -1,17 +1,17 @@
 @prefix : <http://emmo.info/emmo/domain/onto#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix core: <http://www.w3.org/2004/02/skos/core#> .
 @prefix emmo: <http://emmo.info/emmo#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix term: <http://purl.org/dc/terms/> .
```
Comment (jesper-friis) [inline tests/testonto/excelparser/fromexcelonto.ttl:6]: dcterms is the common prefix

File: tests/testonto/excelparser/fromexcelonto.ttl [hunk 2]
```diff
@@ -1,17 +1,17 @@
 @prefix : <http://emmo.info/emmo/domain/onto#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix core: <http://www.w3.org/2004/02/skos/core#> .
```
Comment (jesper-friis) [inline tests/testonto/excelparser/fromexcelonto.ttl:2]: skos is the standard prefix

File: tests/testonto/excelparser/fromexcelonto.ttl (hunk 1 above)
Comment (francescalb) [reply tests/testonto/excelparser/fromexcelonto.ttl:6]: Yes, this is strange. I have no idea why this suddenly happens. I will have to check.

Comment (jesper-friis) [reply tests/testonto/excelparser/fromexcelonto.ttl:6]: I guess it comes from Owlready2 - it uses the second last component in the IRI as a sort of prefix.

File: tests/testonto/excelparser/fromexcelonto.ttl (hunk 2 above)
Comment (francescalb) [reply tests/testonto/excelparser/fromexcelonto.ttl:2]: This needs to be solved in issue 346

**Review by jesper-friis — APPROVED** [review]
Approved. 

The issues with the prefixes will be handled in issue #346

**Pull request merged**